docs/conf/atlantic/2024/news/tickets-grants.rst

```diff
@@ -0,0 +1,67 @@
+
+Get your ticket for the conference
+----------------------------------
+
+**The {{name}} conference will be held online on {{ date.short }}.** The `full event website <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/>`_ is live, with most details, and we'll be filling in more as it gets closer.
+The schedule will run from the
+early afternoon to early evening in most European/African timezones,
+and from early morning to late afternoon in US East Coast/South America timezones.
+
+Tickets are on sale now, so go ahead and `register for your ticket now <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_! Even though our capacity is more flexible due to the virtual event platform, we might still sell out to make sure we're providing a manageable event size for our attendees, as always.
+
+We're going to keep the spirit of our previous virtual conferences:
+
+* Excellent single track of pre-recorded presentations with live moderated Q&A
+* Writing Day sessions
+* Unconference sessions
+* A virtual hallway track
+
+We really want to make sure that folks can get value from being at the event at the same time, and we think it will be another great year of helping our community connect.
+
+
+Grant program
+-------------
+
+If the cost of our tickets is prohibitive to you, you can apply to our Opportunity Grant program.
+As a virtual event, we're excited to be able to offer free tickets to a larger number of people, since there are now minimal other costs to attend the event.
+
+You can apply until **{{ grants.ends }}** on `our website <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_.
+
+
+Submit your proposal
+--------------------
+
+Our `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_ is open.
+Speaking at Write the Docs Atlantic is a fantastic way to share your ideas with our inspiring global community of documentarians.
+If there is something you’d really like to see a talk on, submit a proposal on it, or refer someone else who you think would be good!
+
+**The deadline for submitting proposals is Midnight {{tz}} on {{cfp.ends}}.**
+
+
+Support our community by sponsoring
+-----------------------------------
+
+Every year we are fortunate to have the support of like-minded organizations and communities, who believe in the collaborative and welcoming community spirit and help us make our events great.
+
+We have published our `virtual sponsorship prospectus`_ for the conference,
+and are open for new sponsorship inquiries.
+
+.. _virtual sponsorship prospectus: https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/
+
+We will still run the `job fair <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/job-fair/>`_, helping documentarians and organizations connect with each other. Check out the sponsorship prospectus for more details and to reserve your spot today.
```

Comment:
I wonder about leaving this out, since we haven't had luck filling it the last couple years?

Reply:
I'll turn it into something more generic.